Resources/public/js/alloyeditor/plugins/embed.js

```diff
@@ -86,6 +88,31 @@ YUI.add('ez-alloyeditor-plugin-embed', function (Y) {
                     this._getEzConfigElement();
                     this.setWidgetContent('');
                     this._cancelEditEvents();
+                    this._attachUndoListener();
+                },
+
+                /**
+                 * Attaches event listener for the undo in CKEditor.
+                 *
+                 * @method _attachUndoListener
+                 * @private
+                 */
+                _attachUndoListener: function () {
+                    if (isUndoEventAttached) {
+                        return;
+                    }
+
+                    isUndoEventAttached = true;
+
+                    editor.on('afterCommandExec', function(e) {
```

Comment:
`event` please

Reply:
I want to stay in line with this file and Platform patterns ;)